.woodpecker.yml

```diff
@@ -169,49 +169,36 @@ steps:
       - mv target/debug/lemmy_server target/lemmy_server
     when: *slow_check_paths
 
-  cargo_test:
+  check_diesel_schema:
     image: *rust_image
     environment:
       LEMMY_DATABASE_URL: postgres://lemmy:password@database:5432/lemmy
+      DATABASE_URL: postgres://lemmy:password@database:5432/lemmy
       RUST_BACKTRACE: "1"
       CARGO_HOME: .cargo_home
-      LEMMY_TEST_FAST_FEDERATION: "1"
     commands:
-      - export LEMMY_CONFIG_LOCATION=../../config/config.hjson
-      - cargo test --workspace --no-fail-fast
+      - target/lemmy_server migration --all run
+      - <<: *install_diesel_cli
+      - diesel print-schema --config-file=diesel.toml > tmp.schema
+      - diff tmp.schema crates/db_schema/src/schema.rs
     when: *slow_check_paths
 
-  check_diesel_migration:
```

Comment:
We can't check / test the migrations anymore in CI?

Reply:
This was moved to the unit tests in schema_setup.rs